setup.py

```diff
@@ -42,16 +42,14 @@ def read(fname):
     author='Roy Williams',
     author_email='rwilliams@lyft.com',
     url='https://github.com/lyft/toastedmarshmallow',
-    packages=['toastedmarshmallow', 'marshmallow'],
-    package_dir={
-        'toastedmarshmallow': 'toastedmarshmallow',
-        'marshmallow': 'marshmallow/marshmallow'
-    },
+    packages=['toastedmarshmallow'],
     include_package_data=True,
     extras_require={'reco': EXTRA_REQUIREMENTS},
     license='apache2',
     install_requires=[
-        'attrs >= 17.1.0'
+        'attrs >= 17.1.0',
+        'marshmallow == 3.0.0b2',
```

Comment:
By pinning it to `== 3.0.0b2`, users are forced to use this particular version of marshmallow. Can you change it to a less restrictive version bound?

Reply:
i'd need to make a small patch to upstream to work with the latest v3-line releases to factor out the marshaller again

Reply:
in general for pre-releases, there are no API-compatibility guarantees, so pinning a specific release is safest

Reply:
Ah I see, thanks! I'm interested in using toasted-marshmallow with the upcoming marshmallow 3 as well.

Have you considered forking since this repo seems to be unmaintained?

Reply:
i'd prefer not to do so. i don't understand the core code here well enough to really maintain it.